etc/adminhtml/system.xml

```diff
@@ -171,10 +171,10 @@
                 </group>
                 <group id="omise_offline_paynow" translate="label" sortOrder="160" showInDefault="1" showInWebsite="1" showInStore="1">
                     <label>Enable PayNow QR Payment Solution</label>
-                    <comment>Enables customers to pay using PayNow QR Payment method</comment>
+                    <comment>Enables customers to pay using PayNow QR payment method</comment>
```

Comment:
just minor but not sure if we need `s` at `enables`?

Reply:
Either is correct, but for consistency with the text from other payment methods, I think we should lose the `s`

Reply:
Done